autogen/agentchat/conversable_agent.py

```diff
@@ -1115,10 +1114,17 @@ def my_summary_method(
         return summary
 
     @staticmethod
-    def _last_msg_as_summary(sender, recipient, summary_args) -> str:
+    def _last_msg_as_summary(sender, recipient, summary_args) -> Union[str, List]:
```

Comment:
Changing the return value may affect how carryover is handled? Have you checked `initiate_chats`?

Reply:
The summary might be a list, in this case, so the type propagates to ChatResult.summary and all the downstream users.

Reply:
@jackgerrits for awareness of type change

Reply:
The carryover is now resolved in #2124